recipes/scsplit/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mkdir -p $PREFIX/bin
+mv scSplit/scSplit $PREFIX/bin
```

Comment:
The generated package contains these files:
```
❯ tar xvf scsplit-1.0.8.2-pyhdfd78af_1.tar.bz2 
info/files
info/hash_input.json
info/run_exports.json
info/link.json
info/recipe/build.sh
info/recipe/run_test.sh
info/test/run_test.sh
info/paths.json
info/index.json
info/recipe/meta.yaml.template
info/recipe/meta.yaml
info/recipe/conda_build_config.yaml
info/test/barcodes.tsv
info/recipe/barcodes.tsv
info/about.json
info/test/chr21.bam.bai
info/recipe/chr21.bam.bai
info/recipe/donor_genotype_chr21.vcf
info/test/donor_genotype_chr21.vcf
info/git
info/test/chr21.bam
info/recipe/chr21.bam
python-scripts/scSplit
```
although the executable is moved to `$PREFIX/bin` ...

Reply:
I believe these are the test data and all the output of the test run. scSplit itself doesnt provide any function like `-v` or `-h` for fast tests.

Reply:
With my comment above I tried to say that the generated package has no `bin/` folder at all.
There is `python-scripts/` folder with `scSplit` file inside it but this does not correspond to the logic in `build.sh` ...

Reply:
Ah I see. Maybe I did something wrong here. scSplit is a package that can be installed via pip.  it consists of actually only a single script which is the single `python-scripts/scSplit` file [here](https://github.com/jon-xu/scSplit/blob/master/scSplit). we need to run everytime`<PATH>/scSplit <command> <args>` or `python <PATH>/scSplit <command> <args>` with `PATH` pointing to the location of python site packages. I want to get rid of the `PATH` here. So i moved it to `bin`. Is there any other way to solve this problem? The method itself doesnt have any entrypoint that i can use.